build.gradle

```diff
@@ -37,36 +42,24 @@ task clean(type: Delete) {
     delete rootProject.buildDir
 }
 
-/*configure(subprojects.findAll { !it.name.contains("testpack") && !it.name.startsWith("module") }) {
+configure(subprojects.findAll { !it.name.contains("testpack") && !it.name.startsWith("module") && !it.name.contains("android")}) {
```

Comment:
I'm not eager to reactive this section - I'm trying to avoid having the top-level build.gradle apply settings into other projects because I feel it confuses things.

Reply:
Plenty fair 👍 I had a nicer setup in mind a la Terasology where you just include suitable template build files, but that would take a bit longer and wasn't needed just yet. Will clean this up for sure :-)